lib/action_client/parser_error.rb

```diff
@@ -0,0 +1,3 @@
+module ActionClient
+  ParserError = ParseError
```

Comment:
Is this alias here to help users of the library who forget the name?

It doesn't seem to be referenced anywhere else.

Reply:
Yup! From the last commit message:

> To pre-emptively counteract future typos, this commit also declares
> ActionClient::ParserError as an alias for ActionClient::ParseError.